_health-care/_js/components/insurance-information/InsuranceInformationSection.jsx

```diff
@@ -1,58 +1,63 @@
 import React from 'react';
 
+import ErrorableCheckbox from '../form-elements/ErrorableCheckbox';
+import ErrorableSelect from '../form-elements/ErrorableSelect';
+import ErrorableTextInput from '../form-elements/ErrorableTextInput';
+import { countries, states } from '../../utils/options-for-select';
+
+import Phone from '../personal-information/Phone';
+
 class InsuranceInformationSection extends React.Component {
   render() {
     return (
       <div>
         <div className="row">
           <div className="small-12 columns">
-            <h4>Insurance Information </h4>
```

Comment:
We might need some re-labeling of things. The panel name is "Insurance Information" too. I'll put the `<h4>` back in with modified text calling this "Coverage Information".